lib/cli/index.js

```diff
@@ -56,6 +56,21 @@ module.exports = (options = {}) => {
       silent: 'verbose',
       'log-level': ['silent', 'verbose']
     })
+    .option('canonical', {
+      describe: 'Use canonical support info even when local info is available',
+      type: 'boolean',
+      group: 'preferences'
+    })
+    .option('fetch', {
+      describe: 'Fetch canonical support info from remote repos when necessary',
+      type: 'boolean',
+      group: 'preferences'
+    })
+    .option('base-path', {
```

Comment:
This is intended to be the root of the repo correct?  So you can load support info up the tree?  Just want to make sure I understand.

If so, we might also want to come up with a better `describe` block.  Maybe `Allow support.json files to be loaded from under this base path (default: cwd)`?  Even this is not great, I am just not sure how to describe it well to an end user.

Reply:
@wesleytodd 

This is the root of where you want to allow local support info files to be loaded from.

By default you can't load a file outside in directories above the directory with the package.json. For a since module repo that may be ok, but for monorepos or packages that share a common support-info.json you need to set this if you want to load them locally.

Maybe "Set top most directory from which local support info files can be read. Defaults to same directory in which package.json is located."

Reply:
I like that description!  I think I understood it correctly, but the new wording makes it much more clear I think.